src/Vehicle/Vehicle.h

```diff
@@ -159,13 +159,15 @@ class VehicleBatteryFactGroup : public FactGroup
     Q_PROPERTY(Fact* current            READ current            CONSTANT)
     Q_PROPERTY(Fact* temperature        READ temperature        CONSTANT)
     Q_PROPERTY(Fact* cellCount          READ cellCount          CONSTANT)
+    Q_PROPERTY(Fact* currentWatts       READ currentWatts       CONSTANT)
```

Comment:
Can we call it 'watts' or 'power' instead of 'currentWatts'. That will make it more aligned with existing names in this group, and I think that the 'current' part of the name here is ambiguous.

Reply:
I was unimaginative at the time.
instantPower make more sense to me, because we can have average, total and others..

Reply:
Ok, that is good.